frontend/src/shared/ui/toast/Toast.stories.ts

```
@@ -0,0 +1,279 @@
+            message="Your changes have been saved successfully!"
+            type="success"
+            :dismissible="true"
+            @dismiss="() => {}"
+          />
+        </div>
+      </div>
+    `,
+  }),
+}
+
+export const Error: Story = {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)